src/app/offers/offers.component.html

```diff
@@ -3,7 +3,7 @@
     <div class="flex flex-col mt-2 justify-center items-center">
       <div class="w-full mx-4">
         <div class="mt-4" *ngFor="let offer of offers">
-          <app-offers-card-large class="cursor-pointer" routerLink="12" [offer]="offer"></app-offers-card-large>
```

Comment:
Stop using cursor-pointer class for things that should be clickable. Use either button or anchor tag with (click) or routerLink and this will make your cursor as a pointer.